### PR TITLE
Link to all constituencies in an election from election names on /posts

### PR DESCRIPTION
**Pull request by mhl**
As requested by Andy Lulham (@andylolz).

### CODE REVIEW

File: candidates/templates/candidates/posts.html [hunk 1]
```diff
@@ -19,7 +19,7 @@
         <h3>{{ role_data.role }}</h3>
         {% for election_data in role_data.elections %}
           {% with election=election_data.election %}
-            <h4>{{ election.name }}</h4>
+            <a href="{% url 'constituencies' election=election.slug %}"><h4>{{ election.name }}</h4></a>
             <ul>
```
Comment (struan) [inline candidates/templates/candidates/posts.html]: I'd put the <a> inside the <h4> because otherwise the link is still black text and not at all obvious.

Comment (andylolz) [inline candidates/templates/candidates/posts.html]: > I'd put the inside the
the `<a>` inside the `<h4>`? 😄

Comment (symroe) [inline candidates/templates/candidates/posts.html]: I agree with this change, but it's interesting to me that it's actually valid HTML5 – previously a block level element couldn't be inside in inline element, but the HTML5 validates passes this.

Comment (symroe) [inline candidates/templates/candidates/posts.html]: (Sorry, I mean I agree with putting the inside the :) )

**Pull request merged**